inc/themes/core.base/frontend/templates/postbit/postbit.twig

```diff
@@ -95,9 +95,19 @@
             <input type="checkbox" class="checkbox post__inline-mod" name="inlinemod_{{ post.pid }}" id="inlinemod_{{ post.pid }}" value="1" {% if post.inlinechecked %}checked="checked"{% endif %}/>
         {% endif %}
 
+        {# Post edit information #}
         {% if post.editedmsg %}
-            <a href="#" class="post__edit-log" title="" id="edited_by_{{ post.pid }}">{{ include('partials/icon.twig', {icon: 'pencil-alt', class: ''}, with_context = false) }}</a>
-            <span class="post__edit-reason">({{ post.editnote|raw }} {{ post.editedprofilelink|raw }}.{% if post.editreason %} <em>{{ lang.postbit_editreason }}: {{ post.editreason }}</em>{% endif %})</span>
+            <span class="post__edit-log tooltip">
+                {{ include('partials/icon.twig', {icon: 'pencil-alt', class: ''}, with_context = false) }}
+                <span class="tooltiptext">
+                    {{ post.editnote|raw }} {{ post.editedprofilelink|raw }}
+                    {% if post.editreason %}
+                        <div class="edit-reason">
+                            {{ lang.postbit_editreason }}: <em>{{ post.editreason|raw }}</em>
```

Comment:
Does `post.editreason` need to be `raw`?

Reply:
No, it does not need to use raw; that was a mistake I made.